js/controller/aliasescontroller.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @author Tahaa Karim <tahaalibra@gmail.com>
+ *
+ * ownCloud - Mail
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+define(function(require) {
+	'use strict';
+
+	var $ = require('jquery');
+	var _ = require('underscore');
+	var Radio = require('radio');
+	var AliasesService = require('service/aliasesservice');
+
+	/**
+	 * @param {Account} account
+	 * @returns {undefined}
+	 */
+	function loadAliases(account) {
+		var fetchingAliases = AliasesService.getAliasesEntities(account);
```

Comment:
please use the Backbone Radio for communication with the service (as we do with any other entity)